packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -611,6 +629,51 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
     return textValue;
   };
 
+  /* @conditional-compile-remove(mention) */
+  // Adjust the selection range based on a mouse / touch interaction
+  const handleOnMove = useCallback(
+    (event) => {
+      // Should we do anything?
+      if (interactionStartPoint !== undefined) {
+        // And did selection change?
+        if (
+          event.currentTarget.selectionStart !== null &&
+          (event.currentTarget.selectionStart !== targetSelection?.start ||
+            event.currentTarget.selectionEnd !== targetSelection?.end)
+        ) {
+          let targetStart = event.currentTarget.selectionStart;
+          let targetEnd = event.currentTarget.selectionEnd;
```

Comment:
nitpick: I think we can move these declarations outside the `if` might make the if a little more managable / easier to read

Reply:
Usually, you want to keep your variables as scoped as you can.
I don't really see why you'd want to move these outside the `if` statement?

Reply:
Except to simplify the if statement. Moving